BRANDING.MD

```diff
@@ -75,17 +75,11 @@ Suggested templates:
 * Don't use the Knative name or logo to indicate that your design or
   implementation is compatible with Knative without prior written permission. 
 
-The Knative Trademark Committee understands the community desire for a
-conformance standard, and we do plan to implement one at a future date. We
-also recognize that conformance is a complicated issue with many variables,
-and it is still early days for the Knative project. For comparison, the
-Kubernetes project did not implement the [Certified Kubernetes Conformance
-Program](https://github.com/cncf/k8s-conformance) until version 1.7. Our goal
-is to have a Conformance Working Group created soon.
+For details on the conformance, see [Qualifying Offerings and Self-Testing](https://github.com/knative/specs/blob/main/knative-conformance.md) and [Knative Conformance Certification](https://github.com/knative/specs/blob/main/docs/conformance_procedure.md).
 
 The Knative Trademark Owner and the Knative Trademark Committee reserve the
```

Comment:
This document needs to be updated to clarify who is the current Owner (ie CNCF). this can be done at the top of the doc

Reply:
Owner of what? Of the trademark? If yes, then doesn't the change above (clarifying that the TM is owned by the CNCF) address that?